flybywiresim/a32nx
# chore: small changes after msfs sdk update

## Summary of Changes

1. Remove custom simvar override for msfs sdk instruments
1. Use GenericDataListener as a more performant way to sync
2. 
## Screenshots (if necessary)



## References





## Additional context



Discord username (if different from GitHub):

## Testing instructions

1. Check if PFD works normally (no extensive tests needed, just check it turns on and updates)
2. Load a route and verify that all vectors are properly transferred to the ND (== green line shows on ND as expected)


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, click on the bottom **PR** tab
1. Click on the **A32NX** download link at the bottom of the page
